widgetry/src/drawing.rs

```diff
@@ -246,7 +246,7 @@ pub struct Prerender {
     pub(crate) inner: PrerenderInnards,
     pub(crate) assets: Assets,
     pub(crate) num_uploads: Cell<usize>,
-    pub(crate) scale_factor: RefCell<f64>,
```

Comment:
Unrelated cleanup while I was squinting carefully at this code again. We don't listen to scale factor changed events from winit (which might happen if you drag a window between monitors), so this value doesn't ever change after init.